src/lib.rs

```diff
@@ -320,6 +316,51 @@ impl<T> Stowaway<T> {
         storage as *mut ()
     }
 }
+// These tests should fail miri test
```

Comment:
Change to: "These tests detect undefined behavior in miri under naive, incorrect implementations of Stowaway"